### DISCUSSION TITLE
Legacy makro $(s=:) - to be discontinued.

### DISCUSSION THREAD

**Discussion by jerzyjamroz** [epics-modules/mrfioc2#193] [Ideas]
Let's revise if ```$(s=:)``` is needed and see how many community members use it.

The proposal is to discontinue this macro and go to release 3.0.0.

Let's vote and see the results.

**Comment by gabrielfedel** [comment]
From this PR https://github.com/epics-modules/mrfioc2/pull/194 I have the impression that APS uses this macro, could you confirm @hongran ?

**Reply by hongran** [reply]
Yes, in order to build the db files currently used at APS, I do need this macro. I need it only in a few cases, and I can find a work-around if it is removed in the future version.

**Comment by zioven** [comment]
@jerzyjamroz I am not sure if you mean to eliminate `:` as a separator in the whole module/databases/templates/gui or just in substitutions files.

I would like to have a version of `MRFIOC2` module that is compatible with existing sites that use the `:` as a separator.

This will provide options for them to move to newer versions of MRFIOC without breaking backwards compatibility. 

Also if a wider community feedback is expected, I propose that community is notified via EPICS tech-talk.
